concourse/tasks/package-integration-test.yaml

```diff
@@ -0,0 +1,25 @@
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/gointegtest
```

Comment:
this task is golang specific, consider renaming task to gointegtest or similar

Reply:
rename to `package-gointegtest.yaml`